tools/azure-pipelines/build-apache-repo.yml

```diff
@@ -73,6 +73,17 @@ stages:
           run_end_to_end: false
           container: flink-build-container
           jdk: 8
+      - template: jobs-template.yml
+        parameters:
+          stage_name: ci-adaptive-scheduler
+          test_pool_definition:
+            name: Default
+          e2e_pool_definition:
+            vmImage: 'ubuntu-20.04'
+          environment: PROFILE="-Penable-adaptive-scheduler"
```

Comment:
Shall we add the `-Dflink.hadoop.version=2.10.2` in the profile to keep consistency with other stages? We can also remove the profile from other stages If `2.10.2` is the default value.

Reply:
tbh, I don't know why CI doesn't pick up this PR. The profile configuration will be aligned with the migration to GitHub Actions ([FLIP-360](https://cwiki.apache.org/confluence/display/FLINK/FLIP-396%3A+Trial+to+test+GitHub+Actions+as+an+alternative+for+Flink's+current+Azure+CI+infrastructure)). I don't put any effort into Azure Pipelines anymore. :innocent: